traffic_ops/traffic_ops_golang/logs/log.go

```diff
@@ -79,6 +78,35 @@ func Get(w http.ResponseWriter, r *http.Request) {
 	}
 }
 
+// Get is the handler for GET requests to /logs V4.0.
+func Getv40(w http.ResponseWriter, r *http.Request) {
+	inf, userErr, sysErr, errCode := api.NewInfo(r, nil, []string{"days", "limit"})
+	if userErr != nil || sysErr != nil {
+		api.HandleErr(w, r, inf.Tx.Tx, errCode, userErr, sysErr)
+		return
+	}
+	defer inf.Close()
+	days := DefaultLogDays
+	if pDays, ok := inf.IntParams["days"]; ok {
+		days = pDays
+	}
+
+	a := tc.Alerts{}
+	setLastSeenCookie(w)
+	logs, count, err := getLogV40(inf, days)
+
+	if err != nil {
+		a.AddNewAlert(tc.ErrorLevel, err.Error())
+		api.WriteAlerts(w, r, http.StatusInternalServerError, a)
+		return
+	}
+	if a.HasAlerts() {
```

Comment:
this will never be the case. The only code path that adds an alert writes it and then returns, so this will never evaluate to true

Reply:
Fixed.